critiquebrainz/frontend/views/review.py

```diff
@@ -382,9 +378,6 @@ def edit(id):
         "entity_type": review["entity_type"],
         "review": review,
     }
-    if not review["is_draft"]:
```

Comment:
IIUC, this PR currently doesn't show users the option to upgrade from CC SA v3 to CC SA v4. I don't remember whether we decided the move should be opt-in or mandatory but in any case a dialog should be shown to the user.  Also, even when doing that need to confirm with @alastair whether CC NC can be upgraded to CC SA or not and accordingly a check may be needed.

Till that is implemented in this or a future PR, we should keep this if check.

Reply:
We decided to not worry about asking to upgrade. We'll leave old ones as the current license. If we find that people are asking to upgrade their licenses, the we can add the feature